lib/Swagger/src/ApiClient.php

```diff
@@ -241,7 +241,19 @@ public function callApi($resourcePath, $method, $queryParams, $postData, $header
 
         // Handle the response
         if ($response_info['http_code'] == 0) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "http_code" 7 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)